front/src/applications/operationalStudies/components/Project/AddOrEditProjectModal.tsx

```diff
@@ -67,14 +68,13 @@ export default function AddOrEditProjectModal({ editionMode, project, getProject
       setDisplayErrors(true);
     } else {
       try {
-        let result;
         if (currentProject.image) {
-          const { image, ...currentProjectWithoutImage } = currentProject;
-          result = await post(PROJECTS_URI, currentProjectWithoutImage);
-          await patchMultipart(`${PROJECTS_URI}${result.id}/`, { image });
-        } else {
-          result = await post(PROJECTS_URI, currentProject);
+          const { document_key: docKey } = await post(`${DOCUMENT_URI}/`, currentProject.image, {
+            headers: { 'Content-Type': 'multipart/form-data' },
+          });
```

Comment:
Can you use RTK please ? :)

Reply:
Yes but OpenApi has not been updated yet